src/form.js

```diff
@@ -1,7 +1,7 @@
 export function validateRequiredFields(values, fields, errors) {
     fields.forEach((field) => {
         if (!values[field]) {
-            errors[field] = 'Required field';
+            errors[field] = `msg_error_${field}_required`;
```

Comment:
Not sure if thats okey :) Because there is some projects without translations :)

Reply:
I guess we'll have to add this in our project as separate function then :/ will close the PR